### PR TITLE
Nwd nnw

### PR DESCRIPTION
**Pull request by gyrydymptum**
the homework is done, we are starting the weekend

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @gyrydymptum

**Comment by ziobron** [comment]
⏱ 3 XP extra for @gyrydymptum for on-time delivery

**Review by ziobron — APPROVED** [review]
Formatting is subject to change here. Please read this style guideline - https://github.com/lefticus/cppbestpractices/blob/master/03-Style.md

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
-    return -1;
+if (lhs<0) lhs=-lhs;
+    if (rhs<0) rhs=-rhs;
+      if (lhs==0) return rhs;
+      else if (rhs==0) return lhs;
+      else {
+        while( lhs != rhs )
+        if( lhs < rhs ) rhs -= lhs; 
+        else lhs -= rhs;
+        }return lhs;
```
Comment (Lukasz-334) [inline homework/nwd-nnw/nwdNww.hpp:12]: Better to apply modulo operation, this version is more efficient.

**Comment by ziobron** [comment]
Thanks @Lukasz-334 for Code Review.
🏅 1 XP granted

**Pull request closed without merging**